File: cash.py
import cs50


def calculate_change(coin, money):
    numberOfCoins = 0
    while money >= coin:
        money -= coin
        numberOfCoins += 1
    return numberOfCoins, money


while True:
    change = cs50.get_float("Change owed: ")
    if change > 0:
        break
cents = change * 100
numberOfCoins = 0
listOfCoins = [25, 10, 5, 1]
for coin in listOfCoins:
    tempNofCoins, cents = calculate_change(coin, cents)
    numberOfCoins += tempNofCoins
print(numberOfCoins)

File: cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float change = 0;
    do
    {
        change = get_float("Change owed: ");
    }
    while (0 >= change);
    int cents = lround(change * 100);
    int numberOfCoins = 0;
    while (cents >= 25)
    {
        cents -= 25;
        numberOfCoins++;
    }
    while (cents >= 10)
    {
        cents -= 10;
        numberOfCoins++;
    }
    while (cents >= 5)
    {
        cents -= 5;
        numberOfCoins++;
    }
    while (cents >= 1)
    {
        cents -= 1;
        numberOfCoins++;
    }
    printf("%i\n", numberOfCoins);
}